src/runtime/crt/aot_executor_module/aot_executor_module.c

```diff
@@ -176,7 +176,7 @@ static const TVMBackendPackedCFunc aot_executor_registry_funcs[] = {
 };
 
 static const TVMFuncRegistry aot_executor_registry = {
-    "\x0aget_input\0"
+    "\x0a\0get_input\0"
```

Comment:
Do you need to update src/runtime/crt/graph_executor_module/graph_executor_module.c too?

Reply:
good catch, done

Reply:
I think this [line](https://github.com/apache/tvm/blob/813136401a11a49d6c15e6013c34dd822a5c4ff6/python/tvm/micro/func_registry.py#L44) in `python/tvm/micro/func_registry.py` can be replaced with the following lines.


```
 import struct
 encoded_NumFuncs = [f"\\{v:03o}" for v in struct.pack("H", len(funcs))]
 encoded_funcs = "".join(encoded_NumFuncs) + "\\0".join(funcs)
```